core/src/main/res/values/strings.xml

```diff
@@ -527,9 +527,13 @@
     <string name="pref_delete_removes_from_queue_sum">Automatically remove an episode from the queue when it is deleted.</string>
     <string name="pref_filter_feed_title">Subscription Filter</string>
     <string name="pref_filter_feed_sum">Filter your subscriptions in navigation drawer and subscriptions screen.</string>
-    <string name="hide_subscriptions_where_counter_is_zero">Hide if counter is zero</string>
     <string name="no_filter_label">None</string>
     <string name="subscriptions_are_filtered">Subscriptions are filtered.</string>
+    <string name="subscriptions_counter_greater_zero">Counter greater zero</string>
+    <string name="auto_downloaded">Auto downloaded</string>
+    <string name="not_auto_downloaded">Not auto downloaded</string>
+    <string name="kept_updated">Kept updated</string>
+    <string name="not_kept_updated">Not kept updated</string>
```

Comment:
I am not completely happy about the strings because they are pretty long. In German, they will likely need 3 lines in the dialog. @keunes do you have a idea how to make them shorter?

Reply:
Hi, sorry, didn't see it. Can't think of shorter form, unfortunately. (Would only see a whole different system as an alternative, using a single string with yes/off/no option, but that was rejected.)